CHANGELOG.unreleased.md

```diff
@@ -42,6 +42,33 @@ Examples:
 
 -->
 
+- Config: Support shared configurations ([#5963] by [@azz])
+
+  Sharing a Prettier configuration is simple: just publish a module that exports a configuration object, say `@company/prettier-config`, and reference it in your `package.json`:
```

Comment:
Would it make sense to have an example of such module? So people could basically copy paste and take it from there. Maybe a demo repo with all the default settings under @prettier?

Reply:
I just published my own, may as well use that one 😄 

https://github.com/azz/prettier-config